base/deprecated.jl

```diff
@@ -144,6 +144,8 @@ end
 @deprecate chol(A::Number, ::Type{Val{:L}})         ctranspose(chol(A))
 @deprecate chol(A::AbstractMatrix, ::Type{Val{:L}}) ctranspose(chol(A))
 
+@deprecate cumop! scan!
```

Comment:
worth checking whether it's widely used in packages. if not, then maybe we can skip the deprecation. mainly the issue is the `@deprecate` macro exports the old deprecated name, so isn't quite the right tool for deprecating something that was not originally exported